src/views/v2/day/event/description.php

```diff
@@ -9,13 +9,18 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.9
+ * @version TBD
  *
  * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
  *
  * @see tribe_get_event() For the format of the event object.
  */
+$description = tribe_events_get_the_excerpt( $event->ID, wp_kses_allowed_html( 'post' ) );
```

Comment:
are we planning to keep using our tribe template tag here? (this applies to all uses here)

Reply:
We should move this to `$event->excerpt`, do not overwrite the `$event->post_excerpt` since we do some data treatment on this one, and `$event->post_excerpt` is from DB.